lib/commonOpts.ts

```diff
@@ -26,12 +26,18 @@ const i18nKey = 'lib.commonOpts';
 export function addGlobalOptions(yargs: Argv) {
   yargs.version(false);
 
-  return yargs.option('debug', {
-    alias: 'd',
-    default: false,
-    describe: i18n(`${i18nKey}.options.debug.describe`),
-    type: 'boolean',
-  });
+  return yargs
+    .option('debug', {
+      alias: 'd',
+      default: false,
+      describe: i18n(`${i18nKey}.options.debug.describe`),
+      type: 'boolean',
+    })
+    .option('network-debug', {
+      default: false,
+      type: 'boolean',
+      hidden: true,
```

Comment:
I don't see a compelling reason to unhide this.  If we think external users want to use it, I can add some copy and unhide it.  It's realistically more for our benefit